core/src/main/scala/com/databricks/labs/remorph/discovery/SnowflakeTableDefinitions.scala

```diff
@@ -91,12 +94,14 @@ class SnowflakeTableDefinitions(conn: Connection) {
           val tableName = rs.getString("TABLE_NAME")
           val columns = rs
             .getString("DERIVED_SCHEMA")
-            .split("~")
+            .split("‡")
             .map(x => {
-              val data = x.split(":")
+              val data = x.split("§")
               val name = data(0)
               val dataType = getDataType(data(1))
-              ColumnDetail(name, dataType, data(2).toBoolean)
```

Comment:
don't use `ColumnDetail`, add `metadata` field to `StructField` and add comment there, the same way we do it in Spark.

Reply:
changed the StructField to.
case class StructField(name: String, dataType: DataType, nullable: Boolean = true, metadata: Option[Metadata] = None)
The Metadata class i have used is belongs to spark.(org.apache.spark.sql.types.Metadata)
I have made it as Optional field because we dont have to change the current implementation of StructField which has been used in multiple places.